test/jellyfish/resource_manager_test.exs

```diff
@@ -0,0 +1,33 @@
+defmodule Jellyfish.ResourceManagerTest do
+  use ExUnit.Case
+
+  alias Jellyfish.Component.Recording
+  alias Jellyfish.ResourceManager
+
+  setup do
+    recording_path = Recording.get_base_path() |> Path.join("test")
+    report_path = Path.join(recording_path, "report.json")
+
+    File.mkdir_p!(recording_path)
+    File.touch!(report_path)
```

Comment:
You can set the modification time with the touch function, so we don't have to `Process.sleep` later on in the test.